packages/strapi-design-system/src/DatePicker/DatePicker.js

```diff
@@ -132,6 +131,7 @@ DatePicker.propTypes = {
   minDate: PropTypes.instanceOf(Date),
   onChange: PropTypes.func.isRequired,
   onClear: PropTypes.func,
+  parentFieldId: PropTypes.string,
```

Comment:
Neither this.

Reply:
@ronronscelestes Could you take care of this, please? 🙏🏼

Reply:
on it, my bad